authz/client/src/main/java/org/keycloak/authorization/client/resource/ProtectedResource.java

```diff
@@ -167,7 +167,7 @@ public ResourceRepresentation findByName(String name, String ownerId) {
      * @param maxResult the maximum number of resources to retrieve
      * @return an array of strings with the resource ids
```

Comment:
Javadoc is missing the new parameter matchingUri

Reply:
Added javadoc, but review not marked as outdated.